pallets/loans/src/rate_model.rs

```diff
@@ -98,7 +100,7 @@ impl JumpModel {
     pub const MAX_FULL_RATE: Rate = Rate::from_inner(500_000_000_000_000_000); // 50%
 
     /// Create a new rate model
-    pub fn new_model(
+    pub const fn new_model(
```

Comment:
why we add `const` here, could you share some knowledge about it? thanks.

Reply:
Because `MARKET_MOCK` is a constant structure that calls this function -> https://github.com/parallel-finance/parallel/pull/259/files/ec4c27b93c8e4c91437e55df1242f4f03b8054ec#diff-6f262dd703876877438056eed95093b7b37f23a7bc90a1f14f84eee09ef9b20a

Constant structures are evaluated at compile time in contrast to runtime operations like on-the-fly heap memory allocation. In some cases, these constant elements can improve performance but this specific case is mostly about calling `new_model` on `MARKET_MOCK`.

Reply:
Got it, it's good